src/routes/products.js

```diff
@@ -1,69 +1,129 @@
 const express = require("express");
+const bodyParser = require("body-parser");
+let jsonParser = bodyParser.json();
```

Comment:
body parser middleware should be added globally, on express app level. We discussed it previously